swim/member.go

```diff
@@ -123,11 +129,27 @@ type Change struct {
 	Address           string `json:"address"`
 	Incarnation       int64  `json:"incarnationNumber"`
 	Status            string `json:"status"`
+	Tombstone         bool   `json:"tombstone,omitempty"`
```

Comment:
I trust the stdlib, but I don't trust this line to never change. I think if we change anything in the most used endpoint, we should test it.